aten/src/ATen/cpu/vec/vec256/vec256.h

```diff
@@ -266,6 +266,15 @@ inline Vectorized<int8_t> flip(const Vectorized<int8_t> & v) {
   return _mm256_permute2x128_si256(reversed, reversed, 1);
 }
 
+template<>
+inline Vectorized<uint8_t> flip(const Vectorized<uint8_t> & v) {
```

Comment:
It might be worth splitting these `hflip` changes into their own PR because they are fairly uncontroversial.

Reply:
Done here: https://github.com/pytorch/pytorch/pull/90013